### PR TITLE
Feature/help menu updates august 2016

### PR DESCRIPTION
**Pull request by cmurtaugh**
This PR updates our help menu customizations to work with the new (late August 2016) version of the Canvas UI. 

Instead of fully populating the menu using Javascript, this version simply removes the two items that we cannot control via the UI.  All other customizations can be done in the Settings page of the top-level account in Canvas.

### CODE REVIEW

File: js/help_options.js [hunk 1]
```diff
@@ -1,84 +1,26 @@
...
     MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
     var helpMenuObserver = new MutationObserver(function (mutations) {
         mutations.forEach(function (mutation) {
             var $target = $(mutation.target);
-            if ($target.find('#help-dialog-options').length != 0) {
-                $('#help-dialog-options').empty();
-                addNewHelpItems(menuOptions);
-                $('#help-dialog-options').css('display', 'block');
+            if ($target.find('#help_tray').length != 0) {
+                removeUnwantedHelpOptions()
```
Comment (elliottyates) [inline js/help_options.js]: We might want to disconnect the observer at this point, to reduce strain on the browser.
e.g. https://github.com/harvard-canvas-branding/canvas-branding-global/blob/master/js/iframe_resizer.js#L18

**Comment by elliottyates** [comment]
👍

File: js/help_options.js (hunk 1 above)
Comment (cmurtaugh) [reply js/help_options.js]: Good call -- I'll add it.

Comment (cmurtaugh) [reply js/help_options.js]: Actually - it turns out that the contents of the help menu is reloaded each time you open it, so we do need to leave the observer connected.

**Comment by elliottyates** [comment]
Nice work!

**Pull request merged**